djangocms_versioning_filer/templates/admin/filer/folder/directory_table.html

```diff
@@ -127,7 +127,7 @@
                                             <a href="#" onclick="opener.dismissRelatedImageLookupPopup(window, {{ file.grouper_id|unlocalize }}, '{{ file.icons.48|escapejs }}', '{{ file.label|escapejs }}'); return false;"
                                                 title="{% trans 'Select this file' %}">
                                         {% elif has_change_permission %}
-                                            <a href="{{ file.grouper.get_absolute_url }}{% filer_admin_context_url_params %}"
+                                            <a href="{{ file.get_preview_url }}{% filer_admin_context_url_params %}"
```

Comment:
I think you can just revert that to original filer template: https://github.com/divio/django-filer/blob/develop/filer/templates/admin/filer/folder/directory_table.html#L103